config/jest/config.json

```diff
@@ -6,9 +6,10 @@
   "modulePaths": ["src/components", "src/util"],
   "testPathIgnorePatterns": ["<rootDir>/build", "<rootDir>/dist", "<rootDir>/config", "<rootDir>/src/demos"],
   "moduleNameMapper": {
+    "office-ui-fabric-react/lib/": "office-ui-fabric-react/lib-commonjs/",
     "^.+\\.css$": "identity-obj-proxy"
   },
-  "snapshotSerializers": ["enzyme-to-json/serializer"],
+  "snapshotSerializers": ["enzyme-to-json/serializer", "@uifabric/jest-serializer-merge-styles"],
```

Comment:
`merge-styles` dynamically generates class names that can change at any time, and they end up in snapshots, so this serializer expands them to get reliable tests: https://www.npmjs.com/package/@uifabric/jest-serializer-merge-styles